### PR TITLE
Update nessus.rb

### PR DESCRIPTION
**Pull request by sekritskwurl**
Add support for imported nessus scan results to be imported with nessus_db_import.

### CODE REVIEW

**Comment by sekritskwurl** [comment]
This is the first time I have done something like this with git, if I am doing something wrong please let me know.

**Comment by void-in** [comment]
Nothing wrong with your git fu. Travis should be green in a few minutes and then this could be landed without any problem. Thanks for the contribution.

File: plugins/nessus.rb [hunk 1]
```diff
@@ -1017,7 +1017,7 @@ def is_scan_complete(scan_id)
         complete = false
         status = @n.scan_list
         status["scans"].each { |scan|
-        if scan["id"] == scan_id.to_i && scan["status"] == "completed"
+        if scan["id"] == scan_id.to_i && scan["status"] == "completed" || "imported"
```
Comment (wchen-r7) [inline plugins/nessus.rb]: This is supposed to be a boolean evaluation... but if you're writing it this way, this is what happens:

```
2.1.5 :002 > a == "abc" || "123"
 => "123"
```

Do you mean:

```
2.1.5 :003 > a == "abc" || a == "123"
 => false
```

**Comment by sekritskwurl** [comment]
Most likely, when i tested it out it worked, that would explain why it worked, since i never tested it out with a "completed" scan after that I couldn't say whether it would have not worked or not. 

so going off what you are saying it would look more like: 

 if scan["id"] == scan_id.to_i && scan["status"] == "completed" || scan["status"] == "imported"

Correct?

Is there a way to... retract my previous pull? and perhaps modify it to reflect that change?

**Comment by sekritskwurl** [comment]
Maybe there are brackets I am missing somewhere? 

I made it work for my purposes. I was shocked at that, and figured I'd share. I am thankful for your input.

**Comment by wchen-r7** [comment]
@sekritskwurl To update the pull request, here's what you do:

1. git checkout patch-1
2. Make your changes
3. git commit plugins/nessus.rb
4. git push origin patch-1

And then that will push the code to your patch-1 remote branch, which will also automatically update the pull request.

And yeah it looks like it should be:

```ruby
if scan["id"] == scan_id.to_i && (scan["status"] == "completed" || scan["status"] == "imported")
```

A regex would do the trick too. Something like:

```ruby
scan["status"] =~ /completed|imported/
```

**Comment by sekritskwurl** [comment]
I believe that I updated it properly. Let me know. Thank you for your guidance. :)

**Comment by wvu-r7** [comment]
@sekritskwurl: Nice work! Next time, it helps to use a different commit message for every commit. :)

**Pull request merged**